OpenRA.Mods.Common/Traits/Radar/AppearsOnRadar.cs

```diff
@@ -16,8 +16,10 @@
 
 namespace OpenRA.Mods.Common.Traits.Radar
 {
+	[Desc("Provides a signature on the minimap.")]
 	public class AppearsOnRadarInfo : ConditionalTraitInfo
 	{
+		[Desc("Use center position instead of cell footprint.")]
```

Comment:
```suggestion
		[Desc("Use center position instead of occupied cells.")]
```

I think this would be more accurate. 

Though even then "center position" in our codebase refers to WPos where as this is using CPos (Location)